Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -294,10 +290,3 @@
 "captcha-reload" = "Title for button used to request another CAPTCHA to solve";
 "captcha-prompt" = "Text prompting user to solve a CAPTCHA";
 "mwk-empty-title-error" = "Error description for when an operation couldn't be performed since no title was specified.";
-"home-title" = "Title for home interface.";
```

Comment:
Hm, I' m not sure why it's dropping these specific qqqs. Since this is the first sync I've done for iOS, maybe this is a one time issue? I propose we manually revert the file this time and check again next sync. Let me know what you think.

As a side note, we have a test script on the Android side for detecting missing qq/q entries and a couple tests around string format specifiers. It'd be good to add a card for this testing if you don't already have it.